new-log-viewer/src/components/MenuBar/index.css

```diff
@@ -1,6 +1,28 @@
 .menu-bar {
+    z-index: var(--ylv-menu-bar-z-index);
+
     display: flex;
     flex-direction: row;
     align-items: center;
+
     height: var(--ylv-status-bar-height);
+
+    /* stylelint-disable-next-line custom-property-pattern */
+    box-shadow: 0 1px 0 0 var(--joy-palette-neutral-outlinedBorder);
+}
+
+.menu-bar-logo-container {
+    display: flex;
+    justify-content: center;
+    width: 48px;
+    height: var(--ylv-status-bar-height);
```

Comment:
This should be `--ylv-menu-bar-height` although they both have the same value. Similarly, the `height` in `.menu-bar` should be referencing `--ylv-menu-bar-height` instead as well.

@coderabbitai create a bug issue to track this.